.Rbuildignore

```diff
@@ -8,3 +8,4 @@ local_data
 README.md
 Rplots.pdf
 .DS_Store
+.BBSoptions
```

Comment:
Why add to .gitignore ? These are control files for BioC build system, so it'd need to arrive there,

Reply:
I was not sure - `R CMD check` complains about this file. But you're right, I will remove that line again.

Reply:
I removed this in the latest commit.